airbyte-integrations/connectors/source-mysql-v2/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+data:
+  ab_internal:
+    ql: 200
+    sl: 100
+  allowedHosts:
+    hosts:
+      - ${host}
+      - ${tunnel_method.tunnel_host}
+  connectorSubtype: database
+  connectorType: source
+  definitionId: 196a42fc-39f2-473f-88ff-d68b2ea702e9
+  dockerImageTag: 0.0.1
+  dockerRepository: airbyte/source-mysql-v2
```

Comment:
i think v2 is a temporary connector anyway (eventually it will be renamed to replace mysql v1), and for now since it's marked as archived and it won't get published, so I think it should be fine to leave it as it is.